web/routes/api/sources/testWebhook.ts

```diff
@@ -48,7 +48,7 @@ export const handler: Handlers = {
         sigHash,
         abi: abi.json,
         topics: [topic1, topic2, topic3].flatMap((t) => t ? [toBytes(t)] : []),
-        data: data ? toBytes(data) : new Uint8Array(64),
+        data: data ? toBytes(data) : new Uint8Array(32),
```

Comment:
AFAIK, data can be an empty bytes array.

Reply:
currently EventMessage types data as Uint8Array

Reply:
Ah, I meant Uint8Array 😅
Anyways since you found out that viem requires data to be at minimum 32 bytes, I think you can disregard this comment.